components/core/CarouselSidebarSlate.js

```diff
@@ -92,24 +92,24 @@ const STYLES_AUTOSAVE = css`
   position: absolute;
   opacity: 0;
 
-  @keyframes autosave {
+  @keyframes slate-animations-autosave {
```

Comment:
Whats awesome about these animations are they become global, so we can add them like we add other global styles in the future. Again, a really awesome cleanup task for some later time.